cfn/cfn.json

```diff
@@ -12,6 +12,14 @@
         "InstanceKey": {
             "Type": "String",
             "Description": "Name of a pre-created keypair to associate with any instances created"
+        },
+        "Region": {
+            "Type": "String",
+            "Description": "Amazon region where we are launching"
+        },
+        "AccountID": {
```

Comment:
could we use the [pseudo parameter](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html) instead?